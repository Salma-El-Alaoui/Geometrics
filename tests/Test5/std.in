C c1 1 1 1
R r1 1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
OA oa1 c1 r1
OA oa2 c1 r1 l1 pl1
OA oa3 oa1 oa2
OA oa4 l1 pl1 oa1
OA oa2 c1
OA oa5 c1 r1 123
OA oa6
OA 123 c1 r1
EXIT
