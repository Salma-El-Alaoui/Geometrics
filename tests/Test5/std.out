OK
#Element ajoute: c1
OK
#Element ajoute: r1
OK
#Element ajoute: l1
OK
#Element ajoute: pl1
OK
#Element ajoute: oa1
OK
#Element ajoute: oa2
OK
#Element ajoute: oa3
OK
#Element ajoute: oa4
ERR
#Un element geometrique nomme oa2 existe deja
ERR
#Parametres invalides
ERR
#Parametres invalides
ERR
#Parametres invalides
